src/en/space-station-14/round-flow/proposals/ninja.md

```diff
@@ -0,0 +1,61 @@
+# Space Ninja V2
+
+| Designers | Implemented | GitHub Links |
+|---|---|---|
+| Warpzoned, Doru991 | :warning: Partially | TBD |
+
+## Overview
+
+This proposal is aimed towards addressing some issues that Space Ninjas currently possess, as well as adding some extra flavour and other miscellaneous content.</p>
+<p>New abilities have been added, and existing ones have been tailored to better fit with the stealth-based gameplay the Spider Clan so endorses. The Space Ninja now also come with an extra assortment of apparel, both useful and cool-looking.</p>
+<p>The Space Ninja's weapons have gotten a slight nerf, to compensate, new weapons and armament interactions have been added.
+
+## Background
+
+Currently, albeit being an extremely fun mid-round antag role and definitely one of the better ones, Space Ninjas aren't correctly geared towards what they should be the best at: Stealthing - Which is what this document hopes to address, mostly to add some more uniqueness to antagonist roles since a lot of them can be played by just killing everything in your way, and this includes Space Ninjas.
+
+## Spider Clan Code
+
+All Space Ninjas will now have a Spider Clan Code they will have to live by, and this thus affects gameplay substantially by imposing certain restrictions.
+- The already existing one, being the inability to shoot ranged weapons, will be included in this Code.
+- The second restriction is making it so the Space Ninjas can no longer tend to their own wounds - This includes applying topicals, injecting syringes, consuming pills, and drinking liquid chemicals, as it would be preferred that they are not even spotted during their mission. To compensate, Space Ninjas will now have increased passive regeneration as well as faster bloodloss recovery that far exceeds what normal crewmates could ever achieve due to their bio-engineered bodies.
+
+## New Items & Mechanics
+
+### Flavour Additions
```

Comment:
Another possible suggestion is a special Ninja PDA instead of just having an ID card in their pocket willy nilly.

Reply:
> Another possible suggestion is a special Ninja PDA instead of just having an ID card in their pocket willy nilly.

agreed! I was also thinking of making the station pinpointer built into it, it's just that it'd probably be kind of a much bigger hurdle to implement than other more aesthethic-focused apparel like the sheath or satchel, hence why I didn't include it.